src/core/environment.go

```diff
@@ -540,7 +543,7 @@ func (env *EnvironmentType) Processes() (result []*Process) {
 
 		newProcess := &Process{
 			Pid:        pid,
-			Name:       name,
+			Name:       string([]byte(name)),
```

Comment:
Not sure why this change is needed?

Reply:
this makes a copy of the name and puts it in a new variable. I'll remove it